### ISSUE TITLE
Server sending NaN to agents in perception

### ISSUE THREAD

**Issue by netgusto** [ByteArena/backends#232]
```
[agent] [demoagent:latest] {"method":"perception","payload":{"energy":0,"velocity":[0,1.2500000000000002],"azimuth":6.2273596965095654,"vision":[{"tag":"obstacle","nearedge":[52.454069351199976,9.249067674569478],"center":[52.45401354987693,9.249197977405974],"faredge":[52.45395774855389,9.249328280242468],"velocity":[0,0]},{"tag":"obstacle","nearedg
e":[48.117520678213225,8.484417140366633],"center":[45.66799403675298,14.135609471141073],"faredge":[43.218467395292734,19.786801801915512],"velocity":[0,0]},{"tag":"obstacle","nearedge":[43.218467395292656,19.786801801915477],"center":[51.826931745897,24.83557346938334],"faredge":[60.435396096501336,29.884345136851206],"velocity":[0,0]},{"tag":"obst
acle","nearedge":[NaN,NaN],"center":[NaN,NaN],"faredge":[NaN,NaN],"velocity":[0,0]},{"tag":"obstacle","nearedge":[60.435396096501975,29.884345136851525],"center":[55.994109839930985,39.62490420739875],"faredge":[51.552823583359995,49.36546327794597],"velocity":[0,0]},{"tag":"obstacle","nearedge":[-39.921030318247844,129.37212037363
```

And the agent crash:
```
[agent] [demoagent:latest] SyntaxError: Unexpected token N in JSON at position 695
[agent] [demoagent:latest]     at JSON.parse (<anonymous>)
[agent] [demoagent:latest]     at emitOne (events.js:96:13)
[agent] [demoagent:latest]     at Interface.emit (events.js:188:7)
[agent] [demoagent:latest]     at Interface._onLine (readline.js:247:10)
[agent] [demoagent:latest]     at Interface.onGameServerData (/var/app/node_modules/bytearena-sdk-comm/lib/index.js:68:24)
[agent] [demoagent:latest]     at Interface.<anonymous> (readline.js:384:12)
[agent] [demoagent:latest]     at Array.forEach (native)
[agent] [demoagent:latest]     at Socket.ondata (readline.js:107:10)
[agent] [demoagent:latest]     at Interface._normalWrite (readline.js:383:11)
[agent] [demoagent:latest]     at emitOne (events.js:96:13)
```

**Comment by xtuc** [comment]
oops

**Comment by netgusto** [comment]
Fixed by  302093e
